rust/stackable-cockpit/src/platform/release/spec.rs

```diff
@@ -52,10 +55,10 @@ impl ReleaseSpec {
         info!("Installing release");
 
         for (product_name, product) in self.filter_products(include_products, exclude_products) {
-            info!("Installing product {}", product_name);
+            info!("Installing {}-operator", product_name);
```

Comment:
Does the info!() macro expand the string *and* ship it as OpenTelemetry tag?
If not I would prefer the suggestion below
```suggestion
            info!("Installing {product_name}-operator");
```

Reply:
I'm actually not sure. I will read up on it!

Reply:
So basically this will be transformed into an event with a `message` field which uses the format string. Arguments specified after the format string will be used as arguments in said string. They will **not** be recorded as separate fields.

So we can use the suggested syntax.

Reply:
Changed in 1cdab46

Reply:
Alright, many thanks for looking that up!